modules/uhttp/filters_test.go

```diff
@@ -67,10 +67,13 @@ func TestTracingFilterWithLogs(t *testing.T) {
 		response := testServeHTTP(chain, host)
 		assert.Contains(t, response.Body.String(), "filters ok")
 		assert.True(t, len(buf.Lines()) > 0)
+		var spanFinished bool
 		for _, line := range buf.Lines() {
 			assert.Contains(t, line, "traceID")
 			assert.Contains(t, line, "spanID")
+			spanFinished = strings.Contains(line, "Span finished")
```

Comment:
Hmm, so we are checking the last line only? This seems to be an overkill then :)